strongbox-web-core/src/main/java/org/carlspring/strongbox/domain/DirectoryContent.java

```diff
@@ -0,0 +1,37 @@
+package org.carlspring.strongbox.domain;
+
+import java.util.List;
+import javax.inject.Inject;
+import org.carlspring.strongbox.services.FileContentFetcher;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DirectoryContent
+{       
+    @Inject
+    FileContentFetcher fileContentFetcher;
```

Comment:
I think you shouldn't inject anything here

Reply:
So I should create NEW instance of FileContentFetcher ?

Reply:
you shouldn't have `FileContentFetcher` here at all

Reply:
Got you now. I thought this was the DirectoryContentFetcher .